.github/workflows/daily.yml

```diff
@@ -121,8 +107,8 @@ jobs:
       matrix:
         java: [ 11 ]
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/cache@v1
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
```

Comment:
We drop `actions/cache` line in other places, but upgrade it here. Any reason for this difference?

Reply:
Once Quarkus 3.0.0.Final is out next week, we will drop `s4u/setup-maven-action` and move to setup-java with caching defined there. Thus keeping cache config as is for now here.

Reply:
Sorry, I do not get it.
This change changes step ` windows-build-released-dev` (see below in this file) to drop `actions/cache` and use `actions/setup-java`, but doesn't make the same change for `windows-build-released-jvm` step. What is the difference between these steps?

Reply:
Well, `windows-build-released-dev` is not executed, we should probably drop that.

I will update the PR.

Reply:
Updated

Reply:
https://github.com/stCarolas/setup-maven is handling just maven thus cache can be setup in `actions/setup-java@v3`

https://github.com/s4u/setup-maven-action is composite and it handles cache too, that's what you meant ?

Reply:
There will be other changes once Quarkus 3.0.0.Final is out (less than a week now). So if you don't mind I would prefer to have this merged and do the final set of changes (to use latest maven, cache setup with java action) next week.

Reply:
Or we can postpone all these changes by a week to update together with Quarkus 3